src/integrationTest/java/uk/gov/hmcts/reform/blobrouter/data/EnvelopeEventRepositoryTest.java

```diff
@@ -80,4 +81,46 @@ void should_throw_exception_when_trying_to_create_an_event_for_not_existing_enve
     void should_return_empty_list_when_there_are_no_events_for_given_envelope() {
         assertThat(eventRepo.findForEnvelope(UUID.randomUUID())).isEmpty();
     }
+
+    @Test
+    void should_return_events_for_envelope_ids_order_by_created_at() {
+        // given
+        var envelopeId1 = envelopeRepo.insert(new NewEnvelope("c1", "f1", now(), null, Status.CREATED));
+        var envelopeId2 = envelopeRepo.insert(new NewEnvelope("c2", "f2", now(), null, Status.DISPATCHED));
+
+        var event1a = new NewEnvelopeEvent(envelopeId1, EventType.FILE_PROCESSING_STARTED, null, "note 1");
+        var event2a = new NewEnvelopeEvent(envelopeId2, EventType.FILE_PROCESSING_STARTED, null, "note 3");
+        var event2b = new NewEnvelopeEvent(envelopeId2, EventType.DISPATCHED, null, "note 4");
```

Comment:
I wonder if we need to make a minimal pause (like a millisecond) to guarantee different creation data